app/models/external_registry_client/project529_client.rb

```diff
@@ -0,0 +1,82 @@
+class ExternalRegistryClient::Project529Client
+  BASE_URL = ENV.fetch("PROJECT_529_BASE_URL", "https://project529.com/garage")
+  APP_ID = ENV["PROJECT_529_APP_ID"]
```

Comment:
Does it make sense to specify this in the credential model instead? It might make viewing history more useful.

Reply:
Ah thanks -- should have removed that